### PR TITLE
Ciac-2676/detonate url generic enhancments

### PR DESCRIPTION
**Pull request by efelmandar**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-2676)

## Description
As some integrations such as Wildfire have been updated to have a builtin polling mechanism we can simplify the process of detonation to a single task or smaller playbook that doesn't require the use of generic polling
This PR introduces these new detonate URL playbooks, not a new V but that use fromversion for 6.8 and that will support polling commands

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.8.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Documentation

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -10,7 +10,7 @@ python = "^3.8,<3.11"
 defusedxml = "^0.7.1"
 
 [tool.poetry.group.dev.dependencies]
-demisto_sdk = "1.14.5"
+demisto_sdk = {git = "https://github.com/demisto/demisto-sdk.git", rev = "bd1826b53da8f33a69379f610be1006b14c26c50"}
```
Comment (dorschw) [inline pyproject.toml:13]: please make sure to revert this change.

**Review by dorschw — CHANGES_REQUESTED** [review]

File: Packs/CrowdStrikeFalconX/ReleaseNotes/1_2_27.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
 
 #### Playbooks
-##### Detonate URL - CrowdStrike Falcon Intelligence Sandbox
-Changed mode to quiet.
\ No newline at end of file
+
+##### New: Detonate URL - CrowdStrike Falcon Intelligence Sandbox
+
+- Updated the playbook to use the new command with built-in polling.
```
Comment (altmannyarden) [inline Packs/CrowdStrikeFalconX/ReleaseNotes/1_2_27.md:6]: 1. missing the part with the replace
2. you have two release note files

Comment (altmannyarden) [reply Packs/CrowdStrikeFalconX/ReleaseNotes/1_2_27.md:6]: And by looking at the playbook image - why do we need the playbook?

File: Packs/Palo_Alto_Networks_WildFire/ReleaseNotes/2_1_24.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+
+#### Playbooks
+
+##### New: Detonate URL - WildFire v2.1
+
+- Updated the playbook to use the new command with built-in polling.
```
Comment (altmannyarden) [inline Packs/Palo_Alto_Networks_WildFire/ReleaseNotes/2_1_24.md:6]: 1. 2 release notes again
2. not sure that a playbook is needed

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 4]
```diff
@@ -0,0 +1,1261 @@
...
+    type: condition
+    task:
+      id: a53f7de3-6a8c-4160-8b9d-4b1435672cd9
+      version: -1
+      name: Check if URL exists
+      description: Checks if there is a URL to detonate.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
```
Comment (OmriItzhak) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:634]: What if the URL does not exist?
missing the next task for the 'else' condition

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 5]
```diff
@@ -0,0 +1,1276 @@
...
+      {
+        "position": {
+          "x": 920,
+          "y": 545
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
```
Comment (OmriItzhak) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:819]: change it to skipunavailable: true

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 6]
```diff
@@ -0,0 +1,1276 @@
...
+      {
+        "position": {
+          "x": 30,
+          "y": 545
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
```
Comment (OmriItzhak) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:864]: change it to skipunavailable: true

**Review by OmriItzhak — APPROVED** [review]

**Pull request merged**